x-pack/plugins/security_solution/public/detections/components/user_info/index.test.tsx

```diff
@@ -33,22 +33,21 @@ describe('useUserInfo', () => {
       const { result, waitForNextUpdate } = renderHook(() => useUserInfo());
       await waitForNextUpdate();
 
-      expect(result).toEqual({
-        current: {
-          canUserCRUD: null,
-          hasEncryptionKey: null,
-          hasIndexManage: null,
-          hasIndexMaintenance: null,
-          hasIndexWrite: null,
-          hasIndexUpdateDelete: null,
-          isAuthenticated: null,
-          isSignalIndexExists: null,
-          loading: true,
-          signalIndexName: null,
-          signalIndexMappingOutdated: null,
-        },
-        error: undefined,
+      expect(result.all).toHaveLength(1);
```

Comment:
`result.all` now includes an array of the values if multiple renders were triggered, rather than list the current state over again I just checked that there was only one state over time.